src/creature_states.c

```diff
@@ -158,6 +158,9 @@ short creature_pick_up_spell_to_steal(struct Thing *creatng);
 }
 #endif
 /******************************************************************************/
+//process_state, cleanup_state, move_from_slab, move_check, 
+//override_feed, override_own_needs, override_sleep, override_fight_crtr, override_gets_salary, override_prev_fld1F, override_prev_fld20, override_escape, override_unconscious, override_anger_job, override_fight_object, override_fight_door, override_call2arms, override_follow,
+    //state_type, field_1F, field_20, unsigned short field_21, field_23, unsigned short sprite_idx, field_26, field_27, react_to_cta
```

Comment:
not exactly sure what this comment is supposed to clear up atm it's just a list of values

Reply:
oh yeah probably would've been directly clear if I opened it in my IDE instead of website again so yeah codewise this one is fine by me